app/models/merchant.rb

```diff
@@ -12,4 +12,9 @@ def top_5_customers
     .order(transaction_count: :desc)
     .limit(5)
   end
+
+  def items_not_shipped
+    items.joins(:invoice_items)
```

Comment:
Not a change, just wondering. Would we be able to achieve the same effect as the `#not` method if we did a `!=` operator in the string argument for the `#where` method?